cookbooks/cdo-postfix/attributes/default.rb

```diff
@@ -0,0 +1,7 @@
+default['cdo-postfix'] = {
+  host: 'email-smtp.us-east-1.amazonaws.com',
```

Comment:
Should we be exposing code.org operational details (the smtp host name) in the github repository?  I would have expected this to be a chef secret.